CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM/copy] Only show zstd option when it's supported [#3892](https://github.com/vatesfr/xen-orchestra/issues/3892) (PR [#4326](https://github.com/vatesfr/xen-orchestra/pull/4326))
+- [SDN Controller] Use XAPI `tunnel.status` as described [here](https://xapi-project.github.io/xapi/design/tunnelling.html) (PR [#4322](https://github.com/vatesfr/xen-orchestra/pull/4322))
```

Comment:
Only describe changes of behaviour from a user point of view. A user doesn't need to know which XAPI method you're calling (this can go in the commit message/description) but you can describe changes they might notice instead.